src/event.js

```diff
@@ -120,6 +120,11 @@ jQuery.event = {
 			selector = handleObjIn.selector;
 		}
 
+		// If the selector is invalid, throw any exceptions at attach time
+		if ( selector ) {
+			jQuery.find( selector, elem );
```

Comment:
Also note that this is inside `jQuery.event.add()` and the API surface is `jQuery.fn.on()`. If someone does `$("tr").on("mouseover" ...)` on a table with 100 elements it will check the selector 100 times. The parsed selector is cached and the selection set is small obviously, but is there a downside to moving this up to [function on()](https://github.com/fsateler/jquery/blob/10dc9f515a97c69ac674cfa19298ddd1e5cecbba/src/event.js#L91)?